src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,96 @@
+package duke.task;
+
+import java.util.ArrayList;
+
+/**
+ * Class for storing the taskList which stores the tasks
+ */
+public class TaskList {
+    ArrayList<Task> Tasks;
+
+    //Initializes the taskList
+    public TaskList() {
+        Tasks = new ArrayList<Task>();
+    }
+
+    //Adds a task to the task list.
+    public void addTask(Task task) {
+        Tasks.add(task);
+        System.out.println("Got it! Added: \n" + task);
+        System.out.println("Now you have " + numOfTasks() + " items in your list");
+    }
+
+    //Second add method which does not print any message
+    public void silentAdd(Task task) {
+        Tasks.add(task);
+    }
+
+    //Returns a taskList of the tasks that have matching keyword.
+    public void findTask(String keyword) {
+        TaskList matches = new TaskList();
+        for (Task t : Tasks) {
+            String task = t.getTaskName();
+            String[] taskName = task.split(" ");
+            if (keyword.equals(taskName[0]) || keyword.equals(taskName[1])
+                    || keyword.equals(taskName[2])) {
+                matches.silentAdd(t);
```

Comment:
I like the abstraction of silentAdd() to add your tasks instead of directly adding into the list.